src/main/java/edu/kis/powp/jobs2d/drivers/adapter/LineDrawerAdapter.java

```diff
@@ -0,0 +1,49 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
+
+public class LineDrawerAdapter  extends DrawPanelController implements Job2dDriver {
+    private int startX = 0, startY = 0;
+    private final int type;
+
+    public LineDrawerAdapter(int type) {
+        super();
+        if(type!=1 && type!=2){
+            throw new IllegalArgumentException("Illegal Type : "+ type);
+        }
+        this.type = type;
+    }
+
+    @Override
+    public void setPosition(int x, int y) {
+        this.startX = x;
+        this.startY = y;
+    }
+
+    @Override
+    public void operateTo(int x, int y) {
+        ILine line = LineFactory.getBasicLine();
+
+        if(type == 1){
+            line = LineFactory.getSpecialLine();
+        }
+        if(type == 2){
+            line = LineFactory.getDottedLine();
+        }
```

Comment:
Hardcoded line types (no OCP).
Direct coupling with LineFactory class is unnecessary.

Constants or enums would be preferred (clean code!).